Assets/LeapMotionModules/Attachments/Scripts/Anchors/Anchor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Leap.Unity.RuntimeGizmos;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Attachments {
+
+  public class Anchor : MonoBehaviour, IRuntimeGizmoComponent {
+
+    public float anchorRange = 0.1F;
+
+    public bool IsWithinRange(Vector3 position) {
+      Vector3 delta = this.transform.position - position;
+      return delta.x * delta.x + delta.y * delta.y + delta.z * delta.z < anchorRange * anchorRange;
```

Comment:
this was during my "RAW SPEEEED" phase, I can change it for readability, sometimes I feel compelled to not introduce a function call :P